### ISSUE TITLE
JEI Inventory GUI not appearing...

### ISSUE THREAD

**Issue by MeklonDramicus** [mezz/JustEnoughItems#264]
Hopefully I've missed something _blatantly_ obvious, however I'm struggling to get JEI to work on my current client - I also installed it on 1 other machine with the same mods successfully, the only difference between them being a different shader pack and key bindings.

The mod is showing on the FML list and I can access the mod config ingame, so it's definitely loaded, I just can't access it and it's driving me nuts - so if anyone has any ideas?

MC 1.8.9 // Forge 11.15.1.1855 // JEI 2.28.18.186

Halp?!?

**Comment by shadowfacts** [comment]
Maybe you accidentally hid the overlay? Press Control+O to toggle it.

**Comment by MeklonDramicus** [comment]
Just tried toggling the overlay with the redefined key, no success. Also tried resetting it to default key (just incase of some non-descript conflict) but also no luck.

**Comment by mezz** [comment]
Thanks for the report!  
Can you paste your client log to http://gist.github.com and link it here? 
There's most likely something in the log that will let me figure out what is broken.

**Comment by MeklonDramicus** [comment]
Log as requested... cheers dude!

https://gist.github.com/MeklonDramicus/9a09c4e1a1879cd6263784272023f4bb

**Comment by mezz** [comment]
In your log, JEI never actually starts.

It looks like PixelMon is _seriously_ freaking out about entities spawning.  Since JEI starts when the player "entity" is spawned in the world, I suspect the problem may have to do with PixelMon. (or you never started a world?)

Can you try without PixelMon and report back?

If that doesn't work it may be a Sponge issue, but either way a log without the PixelMon spam would help.

**Comment by MeklonDramicus** [comment]
The world itself is on a dedicated server, somehow I didn't even notice that the Pixelmon entities weren't even spawning _slaps forehead_.

Let me just have a check over the server logs and see what's conflicting -- I knew it would be something obvious :-/

I shall drop another message shortly o>

**Comment by MeklonDramicus** [comment]
Okay, so removed Pixelmon (and a few other depreciated mods) from both Client and Server - still no joy.

Client and server logs >> https://gist.github.com/MeklonDramicus/7a396c83e01f3ec2905349b376d7eb4f#file-fml-client-latest-log-no-pixelmon-mod

Worse case scenario I might just have to try rebuilding the server without sponge, unless, of course, there is a glaringly obvious solution.

**Comment by mezz** [comment]
I think your issue is the same as this one: https://github.com/mezz/JustEnoughItems/issues/157

TLDR that issue:
Remove sponge from the client, it should only be on the server.

**Comment by MeklonDramicus** [comment]
Just confirming removing sponge from the client was a success.

You ser, are Legen..... wait for it.... dary!
